test/e2e/case6_no_ns_test.go

```diff
@@ -64,15 +72,18 @@ var _ = Describe("Test multiple obj template handling", func() {
 			ns2 := utils.GetClusterLevelWithTimeout(clientManagedDynamic, gvrNS,
 				case6NSName2, true, defaultTimeoutSeconds)
 			Expect(ns2).NotTo(BeNil())
-		})
-		It("Cleans up", func() {
+
+			By("Clean up")
```

Comment:
Should this be in an `AfterAll()` function?

Reply:
These specs run as parallel so I added `clean up `to each test